Rapport/sections/appendix/appendix-e/client/userProfile.tex

```diff
@@ -1 +1,24 @@
-\subsubsection{User Profile}
\ No newline at end of file
+\subsubsection{User Profile}
+\begin{figure}[H]
+	\begin{subfigure}{0.80\linewidth}
+		\includegraphics[width=\linewidth]{userManual/client/userProfile/accessUserProfile}
+		\caption{}
+		\label{fig:accessUserProfile}	
+	\end{subfigure}
+	\begin{subfigure}{0.80\linewidth}
+		\includegraphics[width=\linewidth]{userManual/client/userProfile/profile}
+		\caption{}
+		\label{fig:profilePage}
+	\end{subfigure}
+\end{figure}
+
+\begin{userManualItemlist}
+	\item[Step I.] You need to be signed in on your Feide Account to access your user account. 
+	\item[Step II.] Click the button (1) with your Feide name on the navigation bar. (Figure: \ref{fig:accessUserProfile})
+	\item[Step III.] Click "Profile" button (2). (Figure: \ref{fig:accessUserProfile})
+	\item[Step IV.] View user info. The user info contains the Feide username and Feide name. (Figure: \ref{fig:profilePage})
+	\item[Step V.] Click the button (3) labeled "Delete my data" if you want to delete the your user data on the application. This includes your old session data. (Figure: \ref{fig:profilePage})
+	\item[Step VI.] View user stats. The stats includes your total number of sessions, the total amount of questions answered you answered correctly, the total amount of questions you answered incorrectly and how many times you answered "I don't know". (Figure: \ref{fig:profilePage})
```

Comment:
the total amount of questions answered you answered correctly, -> the total amount of questions you answered correctly,